### PR TITLE
Desktop lite: Fix quote error

### PR DESCRIPTION
**Pull request by mdahshan**
Corrected a syntax error in line 414 

log "Executing "\$@\"."

was corrected to

log "Executing \"$@\"."

### CODE REVIEW

**Comment by mdahshan** [comment]
```
/usr/local/share/desktop-init.sh: line 120: unexpected EOF while looking for matching `"'
```

File: src/desktop-lite/install.sh [hunk 1]
```diff
@@ -411,7 +411,7 @@ else
 fi
 
 # Run whatever was passed in
-log "Executing "\$@\"."
+log "Executing \"$@\"."
```
Comment (samruddhikhandale) [inline src/desktop-lite/install.sh:414]: Can you bump the patch version of this Feature? Thanks! 

https://github.com/devcontainers/features/blob/01c0c4b76721aaedfaf450a1d6c0b497eaf6d916/src/desktop-lite/devcontainer-feature.json#L3

**Review by samruddhikhandale — DISMISSED** [review]
Thanks for the quick fix, left one minor comment and we are all set to merge this in!

**Comment by mdahshan** [comment]
Thank you!

File: src/desktop-lite/install.sh (hunk 1 above)
Comment (samruddhikhandale) [reply src/desktop-lite/install.sh:414]: Let me know when it's ready for another review

**Review by samruddhikhandale — APPROVED** [review]

**Pull request merged**